<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>635</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>15777191</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>643</width>
        <height>310</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tab">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="load_tab">
          <attribute name="title">
           <string>Loaded table</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QTableWidget" name="load_table"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="read_table_tab">
          <attribute name="title">
           <string>Read table</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QTableWidget" name="read_table"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="read_graph_tab">
          <attribute name="title">
           <string>Read graph</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QGraphicsView" name="read_graph"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>643</width>
        <height>297</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="1" column="1">
        <widget class="QLineEdit" name="file_dir_Edit"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="read_status_edit"/>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="byte_to_read_edit"/>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="Load_Button">
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="Write_Button">
         <property name="text">
          <string>Write</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLineEdit" name="write_status_edit"/>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="Read_button">
         <property name="text">
          <string>Read</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QPushButton" name="correct_button">
         <property name="text">
          <string>Correct</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="correct_byte_edit"/>
       </item>
       <item row="2" column="1" alignment="Qt::AlignBottom">
        <widget class="QLabel" name="write_status">
         <property name="text">
          <string>Write status</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2" alignment="Qt::AlignBottom">
        <widget class="QLabel" name="read_status">
         <property name="text">
          <string>Read status</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" alignment="Qt::AlignBottom">
        <widget class="QLabel" name="table_dir">
         <property name="maximumSize">
          <size>
           <width>308</width>
           <height>48</height>
          </size>
         </property>
         <property name="text">
          <string>Table file directory</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QLineEdit" name="correct_value_edit"/>
       </item>
       <item row="4" column="1" alignment="Qt::AlignBottom">
        <widget class="QLabel" name="bytes_to_read">
         <property name="text">
          <string>bytes to read</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" alignment="Qt::AlignBottom">
        <widget class="QLabel" name="correct_byte">
         <property name="text">
          <string>correct byte</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2" alignment="Qt::AlignBottom">
        <widget class="QLabel" name="corret_value">
         <property name="text">
          <string>correct_value</string>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>Load_Button</zorder>
      <zorder>Load_Button</zorder>
      <zorder>Write_Button</zorder>
      <zorder>Read_button</zorder>
      <zorder>byte_to_read_edit</zorder>
      <zorder>read_status_edit</zorder>
      <zorder>file_dir_Edit</zorder>
      <zorder>correct_button</zorder>
      <zorder>correct_byte_edit</zorder>
      <zorder>correct_value_edit</zorder>
      <zorder>write_status_edit</zorder>
      <zorder>table_dir</zorder>
      <zorder>write_status</zorder>
      <zorder>read_status</zorder>
      <zorder>bytes_to_read</zorder>
      <zorder>correct_byte</zorder>
      <zorder>corret_value</zorder>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
